src/Lib/Canvas.vala

```diff
@@ -262,12 +307,16 @@ public class Akira.Lib.Canvas : Goo.Canvas {
     private void on_request_change_cursor (Gdk.CursorType? cursor_type) {
         if (cursor_type == null) {
             set_cursor_by_edit_mode ();
```

Comment:
You can leave this here since we needed to handle cursor changes for nob selection, when the edit mode doesn't change, but you should add a check to see if the current cursor is identical to the requested cursor, in order to prevent calling this constantly at every mouse motion.

Reply:
Ok, I think I have to rework the logic in the `HoverManager` in order to emit the event only when the hovered nob changes. 

Right now it triggers way too many events, we could make a flip-flop logic to decrease the event_bus load. 

Thanks for the suggestion, I'll make those changes as soon as I can.